uhh.go

```diff
@@ -28,38 +29,67 @@ func contains(arr []string, str string) bool {
 }
 
 func (u *Uhh) Clone() error {
-	if !gitClone(u.config) {
+	if !gitClone(u.config.Repo(), u.config.LocalRepoPath()) {
 		return fmt.Errorf("Unable to clone")
 	}
 
 	return nil
 }
 
+func (u *Uhh) AddRepo(repoUrl string) error {
+    idx := len(u.config.vals.ReadRepos)
+    repoPath := path.Join(u.config.basePath, strconv.Itoa(idx))
```

Comment:
Shouldn’t we use the function on config here to generate the path? GetReadRepoPath?

Reply:
200 iq